From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 4 Mar 2016 00:59:54 +0100
Subject: [PATCH] Add fuel-agent dependencies to Ubuntu mirror.

Add qemu-user-static and binfmt-support to the list of packages
the local mirror of amd64 Ubuntu should contain.

NOTE: This is needed because fuel-agent from upstream mirror does not
list them as dependencies (not patched for cross-debootstrap).

FIXME: Will building/patching fuel-agent using OPNFV mechanism solve
missing deps from local mirror too?

FIXME: Remove udev package from this patch when Ubuntu Trusty on arm64
fixes persistent device names in the udev it ships out of the box.

FIXME: Remove lshw package from this patch when Ubuntu Trusty on arm64
fixes bug [1]: "AArch64: slow cpuinfo due to redundant loop".

[1] https://bugs.launchpad.net/ubuntu/+source/lshw/+bug/1582181
---
 contrib/fuel_mirror/data/ubuntu.yaml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/contrib/fuel_mirror/data/ubuntu.yaml b/contrib/fuel_mirror/data/ubuntu.yaml
index cd86ec9..2ec4f1d 100644
--- a/contrib/fuel_mirror/data/ubuntu.yaml
+++ b/contrib/fuel_mirror/data/ubuntu.yaml
@@ -77,6 +77,7 @@ packages: &packages
      - "acct"
      - "bash-completion"
      - "bc"
+     - "binfmt-support"
      - "build-essential"
      - "cloud-init"
      - "conntrackd"
@@ -101,6 +102,7 @@ packages: &packages
      - "linux-image-generic-lts-trusty"
      - "live-boot"
      - "livecd-rootfs"
+     - "lshw"
      - "mc"
      - "memcached"
      - "monit"
@@ -116,6 +118,7 @@ packages: &packages
      - "puppet"
      - "python-lesscpy"
      - "python-pip"
+     - "qemu-user-static"
      - "rsyslog-gnutls"
      - "rsyslog-relp"
      - "screen"
@@ -127,6 +130,7 @@ packages: &packages
      - "tmux"
      - "traceroute"
      - "ubuntu-standard"
+     - "udev"
      - "vim"
      - "virt-what"
      - "xinetd"
